src/components/Section/Section.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import cx from 'classnames'
+import styles from 'components/Section/Section.styl'
+
+const Section = props => {
```

Comment:
TODO send this to cozy-ui ?

Reply:
ping @y-lohse @GoOz I do not think we have a component like this in cozy-ui yet.

Reply:
I asked the design team if this is something generic that will land in other apps